Tools/TestWebKitAPI/Tests/WebKitCocoa/FontAttributes.mm

```diff
@@ -30,18 +30,41 @@
 #import "NSFontPanelTesting.h"
 #import "PlatformUtilities.h"
 #import "Test.h"
+#import "TestNavigationDelegate.h"
 #import "TestWKWebView.h"
 #import <WebCore/ColorCocoa.h>
 #import <WebCore/FontCocoa.h>
 #import <WebKit/WKUIDelegatePrivate.h>
+#import <WebKit/WKWebViewPrivate.h>
 #import <cmath>
+#import <pal/spi/cf/CoreTextSPI.h>
 #import <pal/spi/cocoa/NSAttributedStringSPI.h>
 #import <wtf/Vector.h>
 
 #if PLATFORM(IOS_FAMILY)
 #import "UIKitSPI.h"
 #endif
 
+bool _willSetFont = false;
+bool _doneAppleFontTextStyleTest = false;
```

Comment:
It would be cleaner to put these in `AppleFontTextStyleUIDelegate` as ivars in the `@interface`, rather than putting these in the source file.